### ISSUE TITLE
Boot from HD images is not working.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vgmurphy/limbo-android#3]
```
What steps will reproduce the problem?
1. Create a qcow2 image from Limbo
2. Install DSL in the above image
3. Restart VM
4. Booting up fails with Error: Grub Loading. Read Error and VM requires 
restart. Sometimes error is 15,16,17,25. 

What is the expected output? What do you see instead?
Should read boot sector and boot image without errors

Please use labels and text to provide additional information.
When VM is restarted repeatedly it is finally able to boot image.

```

Original issue reported on code.google.com by `ideal2i...@gmail.com` on 10 Sep 2012 at 11:06

**Comment by GoogleCodeExporter** [comment]
```
Workaround:
Set CPU=486 and disable ACPI option.
```

Original comment by `ideal2i...@gmail.com` on 4 Oct 2012 at 1:49

**Comment by GoogleCodeExporter** [comment]
```
This is seems to be a bug with the TSC emulation. Apparently machines with 486 
CPU (which doesn't have TSC) are booting images. For now disabling TSC features 
for all CPU types, for now :)
```

Original comment by `ideal2i...@gmail.com` on 11 Oct 2012 at 4:03

**Comment by GoogleCodeExporter** [comment]
```
Fixed in release 0.9.2
```

Original comment by `ideal2i...@gmail.com` on 11 Oct 2012 at 4:04
- Changed state: **Fixed**
